### ISSUE TITLE
OpenRA Crashes, Fatalar Error, GL Error: 1282

### ISSUE THREAD

**Issue by Bjis** [OpenRA/OpenRA#16122]
System Information
Operating System: Windows (Microsoft Windows NT 6.2.9200.0)
Runtime Version: .NET CLR 4.0.30319.42000
OpenRA Version: release-20181215
Mod: Red Alert, Tiberian Dawn & Dune2000

Additional Information:
1. Installed the OpenRA release-20181215
2. Executed each of the executables for Red Alert, Tiberian Dawn or Dune 2000 via the desktop.
3. Watched how it started and then closed down a few seconds later before I could actually install anyone of them, prompting a fatal error giving me the option to view the logs or view the FAQ.

Note: I have made sure that the antivirus have not interfered in the downloading or installation process of OpenRa. Furthermore, I have made sure that the NVIDIA driver is up to date.

- Logs
[Logs.zip](https://github.com/OpenRA/OpenRA/files/2800286/Logs.zip)

- Print screen of fatal error window that is prompted seconds after having launched the executables
https://i.gyazo.com/66a10ac3befb3bd10d67bc131ffaa768.png

**Comment by abcdefg30** [comment]
This looks like #14211.

**Comment by Bjis** [comment]
Issue has been solved. I did not have to downgrade my driver at all. The issue was caused by the "OSD" feature in "Asus GPU Tweak II". I disabled that and since I have not had any issued with OpenRA, nor is my TeamSpeak or Mirc freezing / crashing anymore. Apparently the OSD feature there interferes with a lot of things on your computer. I heavily suggest that you try to get rid of it, should you have it.
